lib/core/src/Cardano/Wallet/Primitive/AddressDerivation/Sequential.hs

```diff
@@ -347,16 +344,17 @@ deserializeXPrvSeq (k, h) = (,)
   where
     xprvFromText = xprv <=< fromHexText
 
--- | Convert a public key into a hexadecimal string suitable for storing in a
--- text file or database column.
-serializeXPubSeq :: SeqKey purpose XPub -> ByteString
-serializeXPubSeq = toHexText . unXPub . getKey
-
--- | The reverse of 'serializeXPub'. This will fail if the input is not a valid
+-- | The reverse of 'serializeXPub'. This will fail if the input is not
 -- hexadecimal string of the correct length.
 deserializeXPubSeq :: ByteString -> Either String (SeqKey purpose XPub)
 deserializeXPubSeq = fmap SeqKey . (xpub <=< fromHexText)
 
+toHexText :: ByteString -> ByteString
```

Comment:
we should reuse https://github.com/input-output-hk/cardano-wallet/blob/483abd4b920756fbe6605d0a72e37e3e2eb874a9/lib/core/src/Cardano/Wallet/Primitive/AddressDerivation/Common.hs#L37 instead of introducing this one again

Reply:
ah, ok, you removed the common file - sorry only now detected this